tools/create-openbsd-gce-ci.sh

```diff
@@ -54,6 +54,8 @@ cat >etc/rc.local <<EOF
   echo "starting syz-ci"
   fsck -y /dev/sd1a
   mount /syzkaller
+  mount -t mfs -o-s=10G /dev/sd0b /syzkaller/ramdisk
```

Comment:
Don't we need to mkdir first? Mount usually requires that.

Reply:
I tested, I really did, except the persistent disk already had the directory.